app/services/paginate_items_service.rb

```diff
@@ -0,0 +1,24 @@
+class PaginateItemsService < ActiveInteraction::Base
+  integer :per, default: nil
+  integer :page, default: nil
+  boolean :paginate, default: false
```

Comment:
**per** and **page** chotki tams 5, 10 uranq karoxn `nil` inin ?
K aracharkeyi srazu `items.page(page).per(per)` ogtagortis, avelord cheker petkchi.